gui-apps/waybar/waybar-9999.ebuild

```diff
@@ -29,7 +29,6 @@ BDEPEND="
 DEPEND="
 	>=dev-cpp/gtkmm-3.22.0:=
 	dev-libs/jsoncpp:=
-	dev-libs/libinput:=
```

Comment:
Not a dependency:
https://github.com/Alexays/Waybar/commit/74db69dcb7cc9ef46c6036d0474c42e90a47fb82#diff-969b60ad3d206fd45c208e266ccfed38